test/dialect/mysql/test_reflection.py

```diff
@@ -335,10 +335,10 @@ def test_reflection_with_table_options(self, metadata, connection):
         def_table.create(conn)
         reflected = Table("mysql_def", MetaData(), autoload_with=conn)
 
-        if testing.against("mariadb"):
+        if testing.against("mariadb") or testing.db.dialect._is_mariadb:
```

Comment:
the _is_mariadb part here shouldnt be needed, I am testing locally against mdb 10.6 with mariadb connector and it works without it.   just in case there was a reason for tha

Reply:
does `testing.against("mariadb")` returns true if the connection url to connect to mariadb is `mysql://...`?